infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogReplicationAckReader.java

```diff
@@ -172,7 +172,6 @@ private long getTxStreamTail(Map<UUID, Long> tailMap) {
      */
     private long getMaxReplicatedStreamsTail(Map<UUID, Long> tailMap) {
         long maxTail = Address.NON_ADDRESS;
-        replicationContext.refresh();
```

Comment:
any reason for removing this?

Reply:
I thought it was not quite necessary to refresh here - as for SnapshotSync the calculation is based on the config upon reset; and for log entry sync if there are new streams found, it should be captured by the log entry reader

Reply:
if new tables to replicate have been opened, refresh() will be able to find them.  But it is a minor optimization as it will get captured eventually.  It will also not make a difference because the tailMap gets fetched before a refresh so the invocation here has no effect.